### PR TITLE
CRAFTERCMS-3363

### PR DESCRIPTION
**Pull request by dejan-brkic**
Added write global configuration feature

### Ticket reference or full description of what's in the PR
https://github.com/craftercms/craftercms/issues/3363

### CODE REVIEW

File: static-assets/scripts/main.js [hunk 1]
```diff
@@ -1035,11 +1035,11 @@
                     $scope.entities = data.menuItems;
 
                     // TODO: Temp. Remove when global menu does return.
```
Comment (sumerjabri) [inline static-assets/scripts/main.js]: Why not delete this block.

File: static-assets/scripts/main.js [hunk 2]
```diff
@@ -1239,57 +1239,33 @@
         theme: 'ace/theme/textmate',
       });
 
-      // TODO: Temp. Replace the mock-up promise with the actual call - when BE is ready.
```
Comment (sumerjabri) [inline static-assets/scripts/main.js:1242]: Why not delete this block.

File: static-assets/scripts/main.js [hunk 3]
```diff
@@ -1239,57 +1239,33 @@
...
+         })
       ).then((data) => {
-        aceEditor.setValue(data.content || defaultValue);
+        aceEditor.setValue(data.data.content || defaultValue);
         enableUI(true, true);
       });
 
       // Differ the loading of the sample config file to the "background"
       setTimeout(() => {
-        // TODO: Temp. Replace mock-up.
```
Comment (sumerjabri) [inline static-assets/scripts/main.js:1261]: Why not delete this block.

File: static-assets/scripts/main.js [hunk 4]
```diff
@@ -1239,57 +1239,33 @@
...
+           $http.get('/studio/api/1/services/api/1/content/get-content-at-path.bin', {
+             params: { site: 'studio_root', path: '/configuration/samples/sample-studio-config-override.yaml' }
+           })
+        ).then((content) => sampleValue = content.data);
       });
 
       $scope.save = function () {
         enableUI(false);
         const value = aceEditor.getValue();
-        // TODO: Temp. Replace mock-up promise with actual BE call when ready.
```
Comment (sumerjabri) [inline static-assets/scripts/main.js:1287]: Why not delete this block.

**Review by sumerjabri — CHANGES_REQUESTED** [review]

File: static-assets/scripts/main.js (hunk 4 above)
Comment (dejan-brkic) [reply static-assets/scripts/main.js:1287]: already removed (red background and '-')

File: static-assets/scripts/main.js (hunk 3 above)
Comment (dejan-brkic) [reply static-assets/scripts/main.js:1261]: already removed (red background and '-')

File: static-assets/scripts/main.js (hunk 2 above)
Comment (dejan-brkic) [reply static-assets/scripts/main.js:1242]: already removed (red background and '-')

**Review by sumerjabri — APPROVED** [review]

**Review by rart — APPROVED** [review]

**Pull request merged**